src/TwilioVideo.android.js

```diff
@@ -17,6 +17,7 @@ import {
 import React, { Component } from "react";
 
 import PropTypes from "prop-types";
+import { decode, encode } from "base-64";
```

Comment:
This library isn't going to do exactly what you want (or rather, it'll do it in a very annoying to deal with way) because it's base64 decoding the data but then returning it as a string that has to be further decoded (in order to mimic the behavior of old browser APIs that predate the introduction of binary Javascript types like `UInt8Array`). Try https://www.npmjs.com/package/base64-js or https://www.npmjs.com/package/rfc4648 instead.